doc/user/client.md

```diff
@@ -0,0 +1,261 @@
+# Operator-SDK: Controller Runtime Client API
+
+## Overview
+
+The [kubernetes-sigs][org-kubernetes-sigs] organization builds tools to call the [Kubernetes][site-kubernetes] API in a clean, abstracted manner, much like the Operator SDK. The [controller-runtime][repo-controller-runtime] project is meant to help users build [controllers][site-kubernetes-controllers] easily by generating a controller project that interacts with a k8s cluster via CRUD operations. User-defined controller can perform specific tasks in a cluster; [operators][site-operators] can use multiple controllers to perform a variety of tasks. As operators use at least one controller, the SDK can rely on controller-runtime's k8s API code rather than develop a parallel set of API calls to execute the same cluster operations, namely CRUD operations.
+
+controller-runtime defines several interfaces used for cluster interaction:
+- `client.Client`: implementers perform CRUD operations on a k8s cluster.
+- `manager.Manager`: manages shared dependencies, such as Caches and Clients.
+- `reconcile.Reconciler`: compares provided state with actual cluster state and updates the cluster on finding state differences using a Client.
+
+The SDK relies on a `manager.Manager` to create a split `client.Client` interface that performs Create, Update, Delete, Get, and List operations within a `reconcile.Reconciler`'s Reconcile function.
```

Comment:
This line is specific to the default Reconciler client populated by the manger. So you can move it down and consolidate it into the `Default Client` section.